netbout-client/src/main/java/com/netbout/client/RtFriends.java

```diff
@@ -81,8 +85,23 @@ public void invite(final String friend) throws IOException {
             // @checkstyle MultipleStringLiteralsCheck (1 line)
             .body().formParam("name", friend).back()
             .fetch()
-            .as(RestResponse.class)
-            .assertStatus(HttpURLConnection.HTTP_SEE_OTHER);
+            .as(RestResponse.class);
+        if (response.status() == HttpURLConnection.HTTP_MOVED_PERM
+            && response.cookie(RtFriends.COOKIE_RS_FLASH).getValue()
+                .startsWith("incorrect+alias")
+            ) {
+            throw new Friends.UnknownAliasException(
+                response.cookie(RtFriends.COOKIE_RS_FLASH).getValue()
+            );
+        }
+        if (response.status() != HttpURLConnection.HTTP_SEE_OTHER) {
+            throw new AssertionError(
```

Comment:
@dmzaytsev right, but I believe you can call `assertStatus` again:

```java
response.assertStatus(HttpURLConnection.HTTP_SEE_OTHER)
```

instead of having a full `if` statement here.